lib/ex_machina.ex

```diff
@@ -13,7 +13,25 @@ defmodule ExMachina do
     defexception [:message]
 
     def exception(factory_name) do
-      message = "No factory defined for #{inspect factory_name}"
+      message =
+        """
+        No factory defined for #{inspect factory_name}. This may be because you
+        defined a factory with attrs like this: def factory(:my_factory, attrs)
+
+        This has been changed in 0.5.0 to be: def factory(:my_factory). The
+        assoc/2 function has also been removed. Please define your factory
```

Comment:
Maybe this line ("the assoc/3 function has als been removed") to line 29?

"The assoc/3 function has also been removed. belongs_to relationships can now be defined with build:"